newIDE/app/src/Profile/SubscriptionDialog.js

```diff
@@ -33,319 +29,333 @@ import {
-                              ? i18n._(plan.smallDescription)
-                              : ''
-                          }
-                        />
-                        <CardContent>
-                          {plan.descriptionBullets.map(
-                            (descriptionBullet, index) => (
-                              <Column key={index} expand>
-                                <Line noMargin alignItems="center">
-                                  {authenticatedUser.subscription &&
-                                  authenticatedUser.subscription.planId ===
-                                    plan.planId ? (
-                                    <CheckCircle
-                                      style={{
-                                        ...styles.bulletIcon,
-                                        color: muiTheme.message.valid,
-                                      }}
-                                    />
-                                  ) : (
-                                    <CheckCircle style={styles.bulletIcon} />
-                                  )}
-                                  <Text style={styles.bulletText}>
-                                    {i18n._(descriptionBullet.message)}{' '}
-                                    {descriptionBullet.isLocalAppOnly && (
-                                      <Trans>(on the desktop app only)</Trans>
-                                    )}
-                                  </Text>
-                                </Line>
-                              </Column>
-                            )
-                          )}
-                          <Text style={styles.descriptionText}>
-                            {plan.extraDescription
-                              ? i18n._(plan.extraDescription)
-                              : ''}
-                          </Text>
-                        </CardContent>
-                        <CardActions style={styles.actions}>
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <Dialog
+          actions={[
+            <FlatButton
+              label={<Trans>Close</Trans>}
+              key="close"
+              primary={false}
+              onClick={onClose}
+            />,
+          ]}
+          title={<Trans>GDevelop Subscription</Trans>}
+          onRequestClose={onClose}
+          open={open}
+        >
+          <ColumnStackLayout noMargin expand>
+            <Line noMargin alignItems="center">
+              <img src="res/diamond.svg" style={styles.diamondIcon} alt="" />
+              <Text>
+                <Trans>
+                  Get a subscription to gain more one-click exports, remove the
+                  GDevelop splashscreen and messages asking you to get a
+                  subscription. With a subscription, you're also supporting the
+                  development of GDevelop, which is an open-source software
+                  maintained by volunteers in their free time.
```

Comment:
is this message still valid? I'd maybe remove the last part after "open-source software"

Reply:
Indeed, update